docs/conf.py

```diff
@@ -0,0 +1,33 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(".."))
+
+project = 'QE Metrics'
+copyright = '2023, ilausuch'
+author = 'ilausuch'
+release = '1.0.0'
```

Comment:
Ok, we don't have a roadmap, let's say alpha1?

Reply:
i am not sure about that. maybe we can discuss it with Jan in some point

Reply:
I add version as `0-alpha1` and adjust release to reflect the major version